src/util/logger.js

```diff
@@ -78,45 +107,43 @@ async function writeLog(content) {
   }
 }
 
-export async function readLogs() {
+export async function readLogs(type: LogType): Promise<string | void> {
+  const paths = logMap[type]
+
   try {
     let log = ''
-    let exists
 
-    exists = await RNFS.exists(path3)
-    if (exists) {
-      log = log + (await RNFS.readFile(path3))
-    }
-    exists = await RNFS.exists(path2)
-    if (exists) {
-      log = log + (await RNFS.readFile(path2))
-    }
-    exists = await RNFS.exists(path1)
-    if (exists) {
-      log = log + (await RNFS.readFile(path1))
+    for (let i = paths.length - 1; i >= 0; i--) {
+      if (await RNFS.exists(paths[i])) {
+        log = log + (await RNFS.readFile(paths[i]))
+      }
     }
     return log
   } catch (err) {
     global.clog((err && err.message) || err)
   }
 }
 
-export async function log(...info: Array<number | string | null | {}>) {
+export async function logWithType(type: LogType, ...info: Array<number | string | null | {}>): Promise<void> {
   const logs = normalize(...info)
 
   const now = Date.now()
   const d = dateFormat(now, 'HH:MM:ss:l')
 
   try {
     await lock.acquire('logger', async () => {
-      return writeLog(d + ': ' + logs)
+      return writeLog(type, d + ': ' + logs)
     })
   } catch (e) {
     global.clog(e)
   }
   global.clog(logs)
 }
 
+export async function log(...info: Array<number | string | null | {}>): Promise<void> {
```

Comment:
Why not create a type `Log: { type: LogType, data: any }` instead of wrapper functions?

Reply:
I'm just trying to give the equivalent of what we do now with console.log but for the activity log. Less for people to "get used to" when trying to use `logActivity`. Any old `console.log` can also easily be ported to use `logActivity`.